test/integration/scheduler/queue_test.go

```diff
@@ -163,8 +171,14 @@ func TestServiceAffinityEnqueue(t *testing.T) {
 		t.Fatalf("Failed to delete service 'svc': %v", err)
 	}
 
-	// Now we should be able to pop the Pod from activeQ again.
-	podInfo = testCtx.Scheduler.NextPod()
+	// NextPod() is a blocking operation. Wrap it in timeout() to avoid relying on global integration
+	// test timeout (10m) to abort.
```

Comment:
I was trying to search where is the 10m defined, eventually, it seems like it is just a default setting for go testing. :)

>         -timeout d
            If a test binary runs longer than duration d, panic.
            If d is 0, the timeout is disabled.
            The default is 10 minutes (10m).


/lgtm
/hold
hold in case you want a double check from other reviewers, otherwise feel free to lift the label.

Reply:
Thanks for pointing it out. I will update.